internal/pkg/config/environment.go

```diff
@@ -37,3 +37,9 @@ func OverrideFromEnvironment(registry RegistryInfo) RegistryInfo {
 	}
 	return registry
 }
+
+// IsSecurityEnabled determines if security has been enabled.
+func IsSecurityEnabled() bool {
+	env := os.Getenv("EDGEX_SECURITY_SECRET_STORE")
+	return env != "false"
```

Comment:
IMO, from purely usage point of view, there is some issue here: `os.Getenv` cannot distinguish from `empty vaule` and `non-existing`. See [line 98-100](https://golang.org/src/os/env.go) from Golang's comments.    So in this case, `"false"` string and empty string `""` should be treated the same.  Suggest changing to use `os.LookupEnv("EDGEX_SECURITY_SECRET_STORE")` to cover both cases.

Reply:
This logic takes that into consideration. We are assuming security is enabled unless the environment value is present AND `false`. Therefore if the value is empty, non-existent, or blank we will get a non `false` value which we will ignore and treat security as being enabled, which is intended.

Reply:
Using `os.LookupEnv("EDGEX_SECURITY_SECRET_STORE")` would yield the same results just a little more verbose, as we would then inspect the `bool` value first, and if not present we would return false. If it were present we would have to verify that it is `false`. Essentially the same thing with less code.

Reply:
Got cha.  I was not sure that was the assumption and thanks for your explanation.